### ISSUE TITLE
Remote input: adjust receiving buffer size to sample rate

### ISSUE THREAD

**Issue by f4exb** [f4exb/sdrangel#534]
The Remote Input receiving buffer appears to be fixed size resulting in variable delay length. It is ~256k resulting in a 0.5s length in time at 512 kS/s and 10.7s length at 24 kS/s. It appears the 0.5s length is barely enough to keep the R/W balance and 10.7s is a bit of an overkill resulting in an annoying excessively large delay. Something in the area of 2s fixed length in time should be OK.

**Comment by f4exb** [comment]
Not easy to implement. The buffer has conceptually a fixed size depending on nbDecoderSlots which is a constant defined as 16.
`m_framesNbBytes = nbDecoderSlots * sizeof(BufferFrame);`
`BufferFrame` has a fixed size of 64008 bytes (504 * 127)
Thus:
`m_bufferLenSec = nbDecoderSlots * 64008 / 2 * sampleBytes * sampleRate`

So to have constant length in time the `nbDecoderSlots` has te be variable. It would anyway vary in steps as this is an integer and have a minimum value of about 8. These structures in `RemoteInputBuffer` class are depending on `nbDecoderSlots` and thus would have to be dynamically allocated:
```c++
    DecoderSlot          m_decoderSlots[nbDecoderSlots]; //!< CM256 decoding control/buffer slots
    BufferFrame          m_frames[nbDecoderSlots];       //!< Samples buffer
 ```

**Comment by f4exb** [comment]
Was easier than I thought. Buffer sizing is fairly well centralized.

**Comment by f4exb** [comment]
Implemented in v4.14.13 and v5.7.5
